example_config/github-app-auth-hook.py

```
@@ -0,0 +1,314 @@
+
+    for raw_line in sys.stdin:
+        raw_line = raw_line.strip()
+        if not raw_line:
+            continue
+        request_id = None
+        try:
+            request = json.loads(raw_line)
+            if isinstance(request, dict):
+                request_id = request.get("id")
+            response = handle_request(request)
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)